string2.py

```diff
@@ -18,8 +18,12 @@
 
 
 def verbing(s):
-    """Your code goes here.  Edit this docstring."""
-    return
+    if len(s) >= 3 and s[-3:] == "ing":
```

Comment:
Here you could refactor to do the `len(s) >= 3` test from an outer if-statement, then nest the 'ly' test inside